common.sh

```diff
@@ -206,7 +206,12 @@ if [ "${UPSTREAM_IRONIC:-false}" != "false" ] ; then
 fi
 
 if [ -z "$KNI_INSTALL_FROM_GIT" ]; then
-    export OPENSHIFT_INSTALLER=${OPENSHIFT_INSTALLER:-${OCP_DIR}/openshift-baremetal-install}
+    # On 4.16+ the baremetall installer was merged into the regular installer
+    if is_lower_version $OPENSHIFT_VERSION 4.16; then
+        export OPENSHIFT_INSTALLER=${OPENSHIFT_INSTALLER:-${OCP_DIR}/openshift-baremetal-install}
```

Comment:
We also have `OPENSHIFT_INSTALLER_CMD` used in a couple of places, but not here (probably because of dependencies).

I think we need to define a `DEFAULT_OPENSHIFT_INSTALLER_CMD` and set it on the basis of the release version, then use that as the binary name here and as the default for `OPENSHIFT_INSTALLER_CMD` in the 2 places where that is defined.